.devcontainer/Mojo.Dockerfile

```diff
@@ -0,0 +1,160 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Olivier Benz
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The Dockerfile is provided as reference. Please review the redistribution
+# terms of the Mojo license in Section 1 (https://www.modular.com/legal/mojo)
+# prior to distributing pre-built container images.
+# ===----------------------------------------------------------------------=== #
+
+ARG BUILD_ON_IMAGE=glcr.b-data.ch/mojo/base
+ARG MOJO_VERSION=nightly
+
+FROM ${BUILD_ON_IMAGE}:${MOJO_VERSION} as mojo
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG BUILD_ON_IMAGE
+ARG LLVM_VERSION
+
+ENV PARENT_IMAGE=${BUILD_ON_IMAGE}:${MOJO_VERSION} \
+    PARENT_IMAGE_BUILD_DATE=${BUILD_DATE} \
+    LLVM_VERSION=${LLVM_VERSION}
+
+RUN dpkgArch="$(dpkg --print-architecture)" \
+  ## Ensure that common CA certificates
+  ## and OpenSSL libraries are up to date
+  && apt-get update \
+  && apt-get -y install --no-install-recommends --only-upgrade \
+    ca-certificates \
+    openssl \
+## Install Modular
```

Comment:
**Suggestion** I recommend changing this a bit to use `magic` now.  See https://docs.modular.com/magic/ for more info.